helm-chart/renku/templates/ingress.yaml

```diff
@@ -52,12 +52,10 @@ spec:
           serviceName: {{ $keycloakFullname }}-http
           servicePort: {{ $keycloakServicePort }}
       {{- end }}
-      {{- if $gitlabEnabled }}
```

Comment:
how does this work for deployments that dont have GitLab enabled and their path is not `/gitlab`?